microsoft-edge/devtools-guide-chromium/whats-new/includes/edge-whats-new-note.md

```diff
@@ -3,10 +3,10 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.prod: microsoft-edge
 ms.topic: include
-ms.date: 05/13/2021
+ms.date: 05/24/2021
```

Comment:
@Reezaali @zoherghadyali  - Please review this wording, and also the placement of this Include text.

Placement: This Include is placed at top of every What's New page and at top of the What's New firstchild/landing/nav page - remove from nav page, or remove from every What's New page?